frontend/src/components/editor/renderers/layout-select.tsx

```diff
@@ -28,7 +27,7 @@ export const LayoutSelect: React.FC = () => {
 
   // Layouts are not supported in WASM mode by default,
   // unless the feature flag is enabled
-  if (isWasm() && !getFeatureFlag("wasm_layouts")) {
```

Comment:
`wasm_layouts` is still used. since we enabled it in community cloud, but disable it otherwise.